### ISSUE TITLE
Outdated headless app screenshot

### ISSUE THREAD

**Issue by shazron** [AdobeDocs/project-firefly#189]
Issue in /src/pages/resources/cron-jobs/lesson1.md

**Comment by shazron** [comment]
Actually on second thought, the screenshot is accurate, it just doesn't go through all the steps (since the wizard summarizes the steps)

So perhaps the screenshot should show the intermediate steps instead of the final screenshot result

**Comment by purplecabbage** [comment]
yeah, these should probably just be formatted text and not images
